cmd/av/tidy.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"github.com/aviator-co/av/internal/git"
+	"github.com/aviator-co/av/internal/meta"
+)
+
+// tidyDB removes deleted branches from the metadata and returns number of branches removed from the
+// DB.
+func tidyDB(repo *git.Repo, db meta.DB) (int, error) {
+	tx := db.WriteTx()
```

Comment:
Not major, but all of these files (I think) map to actual CLI subcommands (`stack_sync.go` is,,, well,,, `av stack sync`). You could put this in the `actions` package which is our functional catch-all. 🤷

Not something that I think is blocking though.

Reply:
Moved.